PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -28,7 +53,7 @@ The behaviour during individual notification deposit you will see is as follows:
 
 3. If there are fulltext files associated, they will be attached to the eprint
 
-4. The EPrint will be left in the sword user's workarea
+4. The EPrint will be left in the sword user's workarea or review queue, depending on the deposit location that has been selected in PubRouter Admin page.
```

Comment:
Bullet 1  - this is now not quite right & anyway, too technical.  I think we just need to say that:
"The XML supplied by PubRouter will be ingested and used to populate field in the eprint record"

Bullet 2 - should say:
"The supplied XML will be attached as a file to the eprint"

Bullet 4 - For consistency use "eprint" (lower case), and say "...in PubRouter Account page." (not Admin page).